assets/src/dashboard/components/popoverPanel/index.js

```diff
@@ -127,11 +127,11 @@ const PopoverPanel = ({
           <PillFieldset data-testid={'pill-fieldset'}>
             <Legend title={`options for ${title}`} />
             {items.map(
-              ({ label, selected, value, hex, disabled = false }, index) => {
+              ({ label, selected, value, hex, disabled = false }) => {
                 return (
                   <Pill
                     data-testid={'popover-pill'}
-                    key={`${value}_${index}`}
+                    key={value}
```

Comment:
Could this value not be unique here?

Reply:
`value` is used for filtering in places like this, so it needs to be unique to make that work.

<img width="647" alt="Screenshot 2020-10-27 at 10 59 35" src="https://user-images.githubusercontent.com/841956/97286203-80efb280-1843-11eb-93de-112d1bc1f2bb.png">